lib/src/button/Button.tsx

```diff
@@ -31,84 +44,39 @@ const DxcButton = ({
 
   return (
     <ThemeProvider theme={colorsTheme.button}>
-      <DxCButton
+      <Button
         type={type}
-        margin={margin}
         mode={mode !== "primary" && mode !== "secondary" && mode !== "text" ? "primary" : mode}
         disabled={disabled}
-        iconPosition={iconPosition}
-        size={size}
+        aria-disabled={disabled}
+        tabIndex={disabled ? -1 : tabIndex}
         backgroundType={backgroundType}
-        icon={icon}
+        size={size}
+        margin={margin}
+        onClick={() => {
+          onClick();
+        }}
       >
-        <Button
-          type={type}
-          disabled={disabled}
-          disableRipple
-          aria-disabled={disabled}
-          tabIndex={disabled ? -1 : tabIndex}
-          onClick={() => {
-            onClick();
-          }}
-        >
-          {label && iconPosition === "after" && labelComponent}
-          {icon && (
-            <IconContainer label={label} iconPosition={iconPosition}>
-              {typeof icon === "string" ? <ButtonIcon src={icon} /> : icon}
-            </IconContainer>
-          )}
-          {label && iconPosition === "before" && labelComponent}
-        </Button>
-      </DxCButton>
+        {label && iconPosition === "after" && labelComponent}
+        {icon && (
+          <IconContainer label={label} iconPosition={iconPosition}>
+            {typeof icon === "string" ? <ButtonIcon src={icon} /> : icon}
+          </IconContainer>
+        )}
+        {label && iconPosition === "before" && labelComponent}
+      </Button>
     </ThemeProvider>
   );
 };
 
-const sizes = {
-  small: "42px",
-  medium: "120px",
-  large: "240px",
-  fillParent: "100%",
-  fitContent: "unset",
-};
-
-const calculateWidth = (margin, size) => {
-  if (size === "fillParent") {
-    return `calc(${sizes[size]} - ${getMargin(margin, "left")} - ${getMargin(margin, "right")})`;
-  }
-  return sizes[size];
+type ButtonProps = {
+  mode?: "primary" | "secondary" | "text";
```

Comment:
props like mode will always have a value I don´t think it should be optional.

Reply:
Done!